File: Mozu.SiteBuilder.UX.Models/Orders/PaymentReference.cs
using System;
using System.Runtime.Serialization;
using Mozu.SiteBuilder.UX.Models.Customers;

namespace Mozu.SiteBuilder.UX.Models.Orders
{
    [DataContract]
    public class PaymentReference : ModelBase
    {
        [DataMember(Name = "paymentType")]
        public string PaymentType { get; set; }

        [DataMember(Name = "card")]
        public PaymentCardReference Card { get; set; }

        [DataMember(Name = "createDate")]
        public DateTime? CreateDate { get; set; }

        [DataMember(Name = "createBy")]
        public string CreateBy { get; set; }

        [DataMember(Name = "updateDate")]
        public DateTime? UpdateDate { get; set; }

        [DataMember(Name = "updateBy")]
        public string UpdateBy { get; set; }
    }
}

File: Mozu.SiteBuilder.UX.Admin/Scripts/app/model/PaymentReference.js
/**
 * @class Taco.model.PaymentReference
 */
Ext.define('Taco.model.PaymentReference', {
    extend: 'Taco.core.data.Model',

    fields: [{
        "name": "id",
        "type": "int",
        "useNull": true
    }, {
        "name": "paymentType",
        "type": "string",
        "useNull": true
    }, { 
        "name": "card", // Billing contact info under here
        "type": "auto",
        "useNull": true
    }, {
        "name": "createDate",
        "type": "date",
        "useNull": true,
        dateFormat: 'c'
    }, {
        "name": "createBy",
        "type": "string",
        "useNull": true,
        dateFormat: 'c'
    }, {
        "name": "updateDate",
        "type": "date",
        "useNull": true
    }, {
        "name": "updateBy",
        "type": "string",
        "useNull": true
    }],

    proxy: {
        type: 'ajaxproxy',
        api: {
            update: '/admin/app/order/payment/edit'
        },
        reader: {
            type: 'json',
            root: 'items',
            successProperty: 'success',
            messageProperty: "message"
        },
        writer: {
            allowSingle: true,
            type: 'json'
        }
    }
});